From: Emmanuele Bassi <ebassi@gnome.org>
Date: Thu, 4 Oct 2018 14:43:53 +0200
Subject: tests: Update month name check for Lithuanian locale

Update the abbreviated month name in the test to match the GNU libc
translation, coming from CLDR.

Bug: #1562
Applied-upstream: 2.58.2, commit:bf51470adefa7091e685a5df1690f108737664c8
---
 glib/tests/date.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/glib/tests/date.c b/glib/tests/date.c
index b801ca7..ba621e4 100644
--- a/glib/tests/date.c
+++ b/glib/tests/date.c
@@ -365,8 +365,8 @@ test_month_names (void)
       TEST_DATE ( 1,  4, 2018,      "%Y m. %OB",      "2018 m. balandis");
       TEST_DATE ( 1,  5, 2018,      "%Y m. %OB",        "2018 m. gegužė");
       TEST_DATE ( 1,  6, 2018,      "%Y m. %OB",      "2018 m. birželis");
-      TEST_DATE (17,  7, 2018, "%Y m. %b %e d.",     "2018 m. Lie 17 d.");
-      TEST_DATE ( 1,  8, 2018,      "%Y m. %Ob",           "2018 m. Rgp");
+      TEST_DATE (17,  7, 2018, "%Y m. %b %e d.",   "2018 m. liep. 17 d.");
+      TEST_DATE ( 1,  8, 2018,      "%Y m. %Ob",         "2018 m. rugp.");
     }
   else
     g_test_skip ("locale lt_LT not available, skipping Lithuanian month names test");
